lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -260,41 +261,12 @@ def create(work_package, context = {})
                  exec_context: :decorator,
                  getter: -> (*) { datetime_formatter.format_datetime(represented.updated_at) }
 
-        property :status,
-                 embedded: true,
-                 class: ::Status,
-                 decorator: ::API::V3::Statuses::StatusRepresenter
-        property :author,
-                 embedded: true,
-                 class: ::User,
-                 decorator: ::API::V3::Users::UserRepresenter
-        property :responsible,
-                 embedded: true,
-                 class: ::User,
-                 decorator: ::API::V3::Users::UserRepresenter
-        property :assigned_to,
-                 as: :assignee,
-                 embedded: true,
-                 class: ::User,
-                 decorator: ::API::V3::Users::UserRepresenter
-        property :category,
-                 embedded: true,
-                 class: ::Category,
-                 decorator: ::API::V3::Categories::CategoryRepresenter
-        property :priority,
-                 embedded: true,
-                 class: ::IssuePriority,
-                 decorator: ::API::V3::Priorities::PriorityRepresenter
-
         property :activities, embedded: true, exec_context: :decorator
 
         property :version,
                  embedded: true,
-                 exec_context: :decorator
-        property :project,
-                 embedded: true,
-                 class: ::Project,
-                 decorator: ::API::V3::Projects::ProjectRepresenter
+                 exec_context: :decorator,
+                 if: ->(*) { represented.fixed_version.present? }
```

Comment:
@NobodysNightmare I don't understand the `if`. Don't we always display the version property even if it is empty? At least we do it for all the other properties, don't we?